docs/content/concepts/assets/asset-checks.mdx

```diff
@@ -11,292 +11,27 @@ When viewing an asset in Dagster’s UI, you can see all of its checks, and whet
-    },
-    {
-        "name": "items_id_has_no_nulls",
-        "asset": "items",
-        "sql": "select * from items where item_id is null",
-    },
-]
-
-defs = Definitions(
-    assets=[orders, items],
-    asset_checks=[make_check(check_blob) for check_blob in check_blobs],
-    resources={"db_connection": MagicMock()},
-)
-```
+TODO
 
 ---
 
-## Executing checks
-
-### Via the UI
-
-Materializing an asset from the UI will also execute any checks that are defined for that asset. You can also execute checks without materializing the asset from the Checks tab of the asset’s detail page.
-
-### Via sensors and schedules
-
-You can use <PyObject object="define_asset_job" /> to define jobs that execute sets of both assets and checks, and then trigger those jobs via [sensors](/concepts/partitions-schedules-sensors/sensors) or [schedules](/concepts/partitions-schedules-sensors/schedules). By default, checks are included with the assets they check. You can also define jobs that include only checks, or only assets.
-
-```python file=/concepts/assets/asset_checks/jobs.py
-from dagster import (
-    AssetSelection,
-    Definitions,
-    ScheduleDefinition,
-    asset,
-    asset_check,
-    define_asset_job,
-)
-
-
-@asset
-def my_asset(): ...
-
-
-@asset_check(asset=my_asset)
-def check_1(): ...
-
-
-@asset_check(asset=my_asset)
-def check_2(): ...
-
-
-# includes my_asset and both checks
-my_job = define_asset_job("my_job", selection=AssetSelection.assets(my_asset))
-
-
-# includes only my_asset
-my_asset_only_job = define_asset_job(
-    "my_asset_only_job",
-    selection=AssetSelection.assets(my_asset).without_checks(),
-)
-
-# includes check_1 and check_2, but not my_asset
-checks_only_job = define_asset_job(
-    "checks_only_job", selection=AssetSelection.checks_for_assets(my_asset)
-)
-
-# includes only check_1
-check_1_job = define_asset_job("check_1_job", selection=AssetSelection.checks(check_1))
-
-# schedule my_job to run every day at midnight
-basic_schedule = ScheduleDefinition(job=my_job, cron_schedule="0 0 * * *")
-
-defs = Definitions(
-    assets=[my_asset],
-    asset_checks=[check_1, check_2],
-    jobs=[my_job, my_asset_only_job, checks_only_job, check_1_job],
-    schedules=[basic_schedule],
-)
-```
-
----
+## Uses
 
-## Alerting (Dagster Cloud)
+With asset checks, you can:
 
-You can configure alerts on asset checks using [asset based alert policies](/dagster-cloud/managing-deployments/alerts).
+- Guard against inaccurate and stale data by [checking for quality and freshness](/concepts/assets/asset-checks/define-execute-asset-checks)
+- [Execute checks](/concepts/assets/asset-checks/define-execute-asset-checks#executing-checks) using the Dagster UI or schedules/sensors that run jobs with assets and checks
+- Use severity levels and metadata [to include actionable details about check results in the UI](/concepts/assets/asset-checks/define-execute-asset-checks#using-asset-check-results)
```

Comment:
I think also worth adding a bullet that links to the the dbt integration docs on loading dbt tests as asset checks

https://release-1-6-9.dagster.dagster-docs.io/integrations/dbt/reference#loading-dbt-tests-as-asset-checks-